lib/users/get.js

```diff
@@ -0,0 +1,19 @@
+const async = require("async");
+const _ = require("lodash");
+const crypto = require("crypto");
+
+module.exports = function(req, res) {
+    req.pgdb.query("SELECT * FROM users", (error, result) => {
+        if(error) {
+            req.log.error({ error }, "Error while getting users");
+            return res.sendStatus(500);
+        }
+        let users = result.rows.map(row => {
```

Comment:
Yeah, I meant module scope.

It doesn't really change much in this instance, as users is still a mutable object, and the contents can be changed...